app/components/Session/NewSessionForm.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { Button, Switch, Input, Modal, Form, Icon, Row, Col } from 'antd';
+import { remote } from 'electron';
+import FormattedCaps from './FormattedCaps';
+
+const { dialog } = remote;
+const FormItem = Form.Item;
+
+export default class NewSessionForm extends Component {
+
+  getLocalFilePath (success) {
+    dialog.showOpenDialog((filepath) => {
+      if (filepath) {
+        success(filepath);
+      }
+    });
+  }
+
+  getCapsObject () {
```

Comment:
this looks like a reimplementation of `getCapsObject` above, maybe we should reuse instead?